doc/source/getting_started/10min.rst

```diff
@@ -717,6 +711,8 @@ of the columns with labels:
 
 .. ipython:: python
 
+   import matplotlib.pyplot as plt
```

Comment:
I would leave this where it was, and maybe even add a comment (removing the suppress is ok)

Reply:
@jreback Thank you for your comment. I moved the import statement to the previous position and added a comment.